components/treeview/index.css

```diff
@@ -37,8 +37,6 @@
 	--spectrum-treeview-item-background-color-hover: var(--spectrum-gray-200);
 	--spectrum-treeview-item-background-color-focus: var(--spectrum-gray-200);
 
-	--spectrum-treeview-item-border-color-quiet-selected: transparent;
```

Comment:
It looks like this custom property was intended to be used in place of `transparent` in this CSS:
```
.spectrum-TreeView--quiet {
	...
	--mod-treeview-item-border-color-selected: var(--highcontrast-treeview-item-border-color-selected, var(--mod-treeview-item-border-color-selected-quiet, transparent));
}
```
Do you think we should use it there, rather than removing it?

Reply:
Great call! I hadn't seen the direct `transparent` usage — just updated the branch and the changes. ✨